examples/normal_gravity.py

```diff
@@ -0,0 +1,39 @@
+"""
+Normal gravity calculation
+==========================
+
+Gravitational disturbances allows geoscientists to infer the internal structure
+of the Earth. In order to compute a gravitational disturbance, we must define
+a normal gravity.
+Usually, the normal gravity is defined as the gravitational effect of the
+reference ellipsoid. [LiGotze2001]_ introduced a closed-form formula to compute
+the magntiude of the gradient of the gravitational potential generated by any
+reference ellipsoid.
+The :func:`harmonica.normal_gravity` function uses this closed-form formula to
+compute the gravitational effect of the reference ellipsoid given by
+:func:`harmonica.set_ellipsoid`.
+"""
+import numpy as np
+import matplotlib.pyplot as plt
+import cartopy.crs as ccrs
+from harmonica import normal_gravity
+
+# Create a computation grid
+nlat, nlon = 181, 360
+lat = np.linspace(-90, 90, nlat)
+lon = np.linspace(0, 360, nlon)
+lat, lon = np.meshgrid(lat, lon)
```

Comment:
Sound nice to me. From my perspective, it's better to show how the different new libraries of fatiando can be used together. It would also encourage users to explore other libraries than the ones that they mainly need.
I'll get into it.